hawkular-wildfly-agent/src/main/java/org/hawkular/agent/monitor/extension/MonitorServiceConfigurationBuilder.java

```diff
@@ -971,13 +973,16 @@ private static void determineResourceTypeSetDmr(ModelNode config,
                             List<Property> operationList = opModelNode.asPropertyList();
                             for (Property operationProperty : operationList) {
                                 ModelNode operationValueNode = operationProperty.getValue();
-                                String operationName = operationProperty.getName();
+                                String name = operationProperty.getName();
+
+                                List<OperationParam> params = getOpParamListFromOpNode(operationValueNode);
 
                                 PathAddress pathAddress = getPath(operationValueNode, context,
                                         DMROperationAttributes.PATH);
-                                Operation<DMRNodeLocation> op = new Operation<>(ID.NULL_ID, new Name(operationName),
+                                String operationName = getString(operationValueNode, context, DMROperationAttributes.OPERATION_NAME);
+                                Operation<DMRNodeLocation> op = new Operation<>(ID.NULL_ID, new Name(name),
```

Comment:
I renamed operationName to name some lines above as it was more logical to me, but seems to confuse you now :)